README.md

```diff
@@ -205,6 +205,26 @@ Since this project is contained all within one file, `main.py`. You can simply a
               API_BASE_URL: https://wakapi.dev/api
     ```
 
+5. If you don't like to share how much time you spent in each language, you can add `SHOW_TIME: false` in your workflow file like this
+
+```yml
+    - uses: athul/waka-readme@master
+        with:
+          WAKATIME_API_KEY: ${{ secrets.WAKATIME_API_KEY }}
+          SHOW_TIME: false
+```
+
+`SHOW_TIME` flag can be set to false if you don't want to display the time spent in the readme, by default it will be false. Here is an example output with `SHOW_TITLE` set to true.
```

Comment:
```suggestion
`SHOW_TIME` flag can be set to false if you don't want to display the time spent in the readme, by default it will be false. Here is an example output with `SHOW_TIME` set to true.
```

I hope this is what you meant 🙈